exercises/two-bucket/.meta/tests.toml

```diff
@@ -0,0 +1,28 @@
+[canonical-tests]
+
+# Measure using bucket one of size 3 and bucket two of size 5 - start with bucket one
+#"a6f2b4ba-065f-4dca-b6f0-e3eee51cb661" = true
+
+# Measure using bucket one of size 3 and bucket two of size 5 - start with bucket two
+#"6c4ea451-9678-4926-b9b3-68364e066d40" = true
+
+# Measure using bucket one of size 7 and bucket two of size 11 - start with bucket one
+#"3389f45e-6a56-46d5-9607-75aa930502ff" = true
+
+# Measure using bucket one of size 7 and bucket two of size 11 - start with bucket two
+#"fe0ff9a0-3ea5-4bf7-b17d-6d4243961aa1" = true
+
+# Measure one step using bucket one of size 1 and bucket two of size 3 - start with bucket two
+#"0ee1f57e-da84-44f7-ac91-38b878691602" = true
+
+# Measure using bucket one of size 2 and bucket two of size 3 - start with bucket one and end with bucket two
+#"eb329c63-5540-4735-b30b-97f7f4df0f84" = true
+
+# Not possible to reach the goal
+#"449be72d-b10a-4f4b-a959-ca741e333b72" = true
+
+# With the same buckets but a different goal, then it is possible
+#"aac38b7a-77f4-4d62-9b91-8846d533b054" = true
+
+# Goal larger than both buckets is impossible
+#"74633132-0ccf-49de-8450-af4ab2e3b299" = true,
```

Comment:
I think the lines with the SHAs, should not have octothorpes prefixed as it comments out those line

Reply:
Oh got it, thanks!